config/locales/en.yml

```diff
@@ -1187,37 +1187,37 @@ en:
           volcano: "Volcano"
           water: "Water"
           wetland: "Wetland"
-          wood: "Wood"
+          wood: "Woods"
           "yes": "Natural Feature"
         office:
-          accountant: "Accountant"
-          administrative: "Administration"
+          accountant: "Accountant Office"
+          administrative: "Administrative Office"
           advertising_agency: "Advertising Agency"
-          architect: "Architect"
-          association: "Association"
-          company: "Company"
+          architect: "Architect Office"
+          association: "Nonprofit Organization Office"
+          company: "Corporate Office"
           diplomatic: "Diplomatic Office"
           educational_institution: "Educational Institution"
           employment_agency: "Employment Agency"
           energy_supplier: "Energy Supplier Office"
-          estate_agent: "Estate Agent"
+          estate_agent: "Real Estate Agent"
           financial: "Financial Office"
           government: "Governmental Office"
           insurance: "Insurance Office"
           it: "IT Office"
-          lawyer: "Lawyer"
+          lawyer: "Law Office"
           logistics: "Logistics Office"
           newspaper: "Newspaper Office"
           ngo: "NGO Office"
-          notary: "Notary"
+          notary: "Notary Office"
           religion: "Religious Office"
           research: "Research Office"
-          tax_advisor: "Tax Advisor"
-          telecommunication: "Telecommunication Office"
+          tax_advisor: "Tax Advisor Office"
+          telecommunication: "Telecom Office"
```

Comment:
I think that in this case `Telecommunication Office` is just as clear, and better matches the tag